randovania/games/prime2/logic_database/Torvus Bog.txt

```diff
@@ -3747,13 +3747,11 @@ Extra - in_dark_aether: False
   * Dark Door to Training Chamber/Door to Transit Tunnel East; Dock Lock Rando incompatible with: Screw Attack Blast Shield
   * Extra - dock_name: West
   > Door to Catacombs
-      Any of the following:
-          No Gravity Boost
-          All of the following:
-              Morph Ball
-              Any of the following:
-                  Morph Ball Bomb
```

Comment:
Why is this using a `No` here?

Reply:
Because otherwise you need other items

Reply:
I didn't pay attention to the diff, my b.